assets/app/scripts/filters/httpHttps.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+angular.module('openshiftConsole')
+  .filter('httpHttps', function() {
+    return function(input) {
```

Comment:
input seems like the wrong name for this parameter, change it to something like isSecure